requirements.dev.in

```diff
@@ -12,7 +12,9 @@ pytest-cov
 pytest-django
 pytest-freezer
 pytest-playwright
-responses
+# pin responses pending this bug which breaks mypy
+# https://github.com/getsentry/responses/issues/751
+responses<0.25.5
```

Comment:
in the linked ticket the fix will be in responses 0.25.6 - should we wait & upgrade to that & remove this pin?

Reply:
Ah, it wasn't done when I made the PR. I can just update again now.